core/conn/jdbc_type2/src/main/java/org/apache/trafodion/jdbc/t2/SQLMXDesc.java

```diff
@@ -424,7 +424,7 @@ void checkValidNumericConversion(Locale locale) throws SQLException
     public static final int SQLCHARSETCODE_GB2312           = 17;
 
 	public static final String SQLCHARSETSTRING_UNKNOWN     = "UNKNOWN";
-	public static final String SQLCHARSETSTRING_ISO88591    = "ISO88591";
+	public static final String SQLCHARSETSTRING_ISO88591    = "ISO-8859-1";
```

Comment:
I am curious why this has been changed. In the SQL engine itself, we use "ISO88591". Will this cause a problem?